app/controllers/reports_controller.rb

```diff
@@ -80,6 +80,16 @@ def where_did_you_hear_summary
     @report = WhereDidYouHearSummary.new(@where_did_you_hears.results)
   end
 
+  def twilio_numbers
+    @twilio_numbers_report = TwilioNumbersReport.new(twilio_numbers_report_params)
+    if request.post?
```

Comment:
Not a huge fan of actions serving multiple verbs this way. Can we separate them?